java/src/main/java/com/paymentsense/sample/model/AccessTokenRequest.java

```diff
@@ -0,0 +1,70 @@
+package com.paymentsense.sample.model;
+
+// Limited subset of request fields
+// See https://docs.connect.paymentsense.cloud/ConnectE/RestApi for a full list of request parameters
+public class AccessTokenRequest {
+    private String gatewayUsername;
```

Comment:
Oh man, it's ages since I last looked at any Java. Is this still required? There's no shorthand for properties?

Reply:
There are libraries to use annotations instead of writing things out, like lombok.  I haven't used any of them regularly though.  Looks like Java 16 just introduced the concept of records, which might be applicable?  Released just this spring (LTS release last month): https://weakreference.medium.com/java-16-records-f16c2ecb4b05